tests/languages/json/number_feature.test

```diff
@@ -4,6 +4,9 @@
 5.0e8
 0.2E+2
 47e-5
+-1.23
+-2.34E33
+-4.34E-33

```

Comment:
What's up with the `+`?

Reply:
That's a git diff, saying those lines were added :D

Reply:
... Ups.
My bad